.github/workflows/ci.yml

```diff
@@ -1,6 +1,13 @@
 name: Build
 
 on:
+  workflow_dispatch:
+  # The push trigger would run the CI workflow when any changes get merged to main branch.
+  # The build badge on the main README uses the CI results on the main branch to report the build status.
+  push:
+    branches: [ 'main*' ]
+    paths-ignore:
+    - '**.md'
```

Comment:
We could remove the paths-ignore bit. For md-only PRs the CI is smart enough now to just run the lint portions which seem valid enough to run on main as anything else.

Reply:
That's a good point. Thanks!